src/pycram/worlds/bullet_world.py

```diff
@@ -35,10 +33,6 @@ class is the main interface to the Bullet Physics Engine and should be used to s
     manipulate the Bullet World.
     """
 
-    # Check is for sphinx autoAPI to be able to work in a CI workflow
-    if rosgraph.is_master_online():  # and "/pycram" not in rosnode.get_node_names():
```

Comment:
can this be safely removed?

Reply:
Yes, I moved this to the __init__.py of the ros package. So it should be called anytime something wants to interact with ROS.